agent/src/config/configsource.h

```diff
@@ -20,25 +20,25 @@
 #pragma once
 
 #include "softwarecontainer-common.h"
```

Comment:
Nothing in this class needs anything from `softwarecontainer-common.h` (since you no longer declare a log context).